From 0df9a8dae5077864b69781eb5d03fbdfe24bfea3 Mon Sep 17 00:00:00 2001
From: Christophe Chapuis <chris.chapuis@gmail.com>
Date: Sat, 4 Mar 2017 13:46:29 +0000
Subject: [PATCH] Add MediaRecorder flag for libhybris

Change-Id: I531d85308520580a8e92eb31aa540580186a7329
---
 BoardConfig.mk | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/BoardConfig.mk b/BoardConfig.mk
index a9f4b65..630ebad 100644
--- a/BoardConfig.mk
+++ b/BoardConfig.mk
@@ -172,3 +172,6 @@ USE_MINIKIN := true
 
 # Include an expanded selection of fonts
 EXTENDED_FONT_FOOTPRINT := true
+
+# libhybris
+BOARD_HAS_MEDIA_RECORDER_PAUSE := true
-- 
2.7.4

