From c9e923b810bea44008e7fd02a08a5dc5c440fe78 Mon Sep 17 00:00:00 2001
From: Herman van Hazendonk <github.com@herrie.org>
Date: Thu, 30 Nov 2017 09:56:00 +0100
Subject: [PATCH] mido-vendor.mk: Drop APK's that we don't need anyway.

Signed-off-by: Herman van Hazendonk <github.com@herrie.org>
---
 mido/mido-vendor.mk | 12 ------------
 1 file changed, 12 deletions(-)

diff --git a/mido/mido-vendor.mk b/mido/mido-vendor.mk
index dddbe14a..c0b04492 100644
--- a/mido/mido-vendor.mk
+++ b/mido/mido-vendor.mk
@@ -1077,18 +1077,6 @@ PRODUCT_PACKAGES += \
     libtime_genoff \
     libts_detected_face_hal \
     libts_face_beautify_hal \
-    QtiTelephonyService \
-    TimeService \
-    datastatusnotification \
-    embms \
-    fastdormancy \
-    shutdownlistener \
-    CNEService \
-    QtiTetherService \
-    com.qualcomm.location \
-    qcrilmsgtunnel \
-    ims \
-    imssettings \
     com.qti.location.sdk \
     qcrilhook \
     qdcm_calib_data_nt35532_fhd_video_mode_dsi_panel \
-- 
2.13.2.windows.1

