From 7c6a2eb536d9f8409fec62f843fa450eb6640f1c Mon Sep 17 00:00:00 2001
From: Herrie <Github.com@herrie.org>
Date: Thu, 28 Dec 2017 17:23:55 +0100
Subject: [PATCH] athene: Remove packages from athene-vendor.mk

Signed-off-by: Herman van Hazendonk <github.com@herrie.org>
---
 athene/athene-vendor.mk | 20 +-------------------
 1 file changed, 1 insertion(+), 19 deletions(-)

diff --git a/athene/athene-vendor.mk b/athene/athene-vendor.mk
index c63a4651..814e2155 100644
--- a/athene/athene-vendor.mk
+++ b/athene/athene-vendor.mk
@@ -1033,23 +1033,5 @@ PRODUCT_COPY_FILES += \
 PRODUCT_PACKAGES += \
     libril \
     libtime_genoff \
-    TimeService \
-    CNEService \
-    qcrilmsgtunnel \
-    ims \
-    colorservice \
-    imssettings \
-    imscmlibrary \
     qcnvitems \
-    qcrilhook \
-    AttPhoneExt \
-    CarrierServices \
-    CarrierSettingsExt \
-    ConnMO \
-    CoreSettingsExt \
-    DCMO \
-    DiagMon \
-    DMService \
-    SprintDM \
-    DMConfigUpdateLight \
-    atfwd
+    qcrilhook
-- 
2.13.2.windows.1

