From e8861effa74fb660a565db0665749210d7430bc7 Mon Sep 17 00:00:00 2001
From: Herrie <Github.com@herrie.org>
Date: Thu, 28 Dec 2017 16:43:52 +0100
Subject: [PATCH] device.mk: Drop CMActions

Signed-off-by: Herman van Hazendonk <github.com@herrie.org>
---
 device.mk | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/device.mk b/device.mk
index c8635a0..166fa97 100644
--- a/device.mk
+++ b/device.mk
@@ -107,10 +107,6 @@ PRODUCT_PACKAGES += \
     libbson \
     Snap
 
-# CMActions
-PRODUCT_PACKAGES += \
-    CMActions
-
 # Display
 PRODUCT_PACKAGES += \
     gralloc.msm8952 \
-- 
2.13.2.windows.1

