From 5cb2d53d86a6850f921454d252fd5f34aa62f2b5 Mon Sep 17 00:00:00 2001
From: Herrie <Github.com@herrie.org>
Date: Thu, 21 Dec 2017 09:55:00 +0100
Subject: [PATCH] onyx-vendor.mk: Drop APK's that we don't need anyway.

Signed-off-by: Herman van Hazendonk <github.com@herrie.org>
---
 onyx/onyx-vendor.mk | 7 +------
 1 file changed, 1 insertion(+), 6 deletions(-)

diff --git a/onyx/onyx-vendor.mk b/onyx/onyx-vendor.mk
index 1177fd06..c87a44f0 100644
--- a/onyx/onyx-vendor.mk
+++ b/onyx/onyx-vendor.mk
@@ -236,12 +236,7 @@ PRODUCT_COPY_FILES += \
 PRODUCT_PACKAGES += \
     libloc_api_v02 \
     libloc_ds_api \
-    libTimeService \
     libmm-abl \
     libtime_genoff \
-    TimeService \
-    shutdownlistener \
-    qcrilmsgtunnel \
     ConnectivityExt \
-    qcnvitems \
-    qcrilhook
+    qcnvitems 
-- 
2.13.2.windows.1

