From f71810e3ffd1e975d710abf54da337e40b1087a2 Mon Sep 17 00:00:00 2001
From: Christophe Chapuis <chris.chapuis@gmail.com>
Date: Sat, 4 Mar 2017 14:10:03 +0000
Subject: [PATCH] Add MediaRecorder flag for libhybris

Change-Id: I55876b950d1323cd12287b9b7694b1b33211dd09
---
 BoardConfig.mk | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/BoardConfig.mk b/BoardConfig.mk
index 43c8ede..4a31646 100755
--- a/BoardConfig.mk
+++ b/BoardConfig.mk
@@ -133,3 +133,7 @@ TARGET_RECOVERY_PIXEL_FORMAT := "BGRA_8888"
 BOARD_RECOVERY_SWIPE := true
 BOARD_HAS_NO_SELECT_BUTTON := true
 BOARD_USE_CUSTOM_RECOVERY_FONT := \"roboto_15x24.h\"
+
+# libhybris
+BOARD_HAS_MEDIA_RECORDER_PAUSE := true
+
-- 
2.7.4

