From 9cccc78755e11563a2089d6efb069c183a181758 Mon Sep 17 00:00:00 2001
From: Herrie <Github.com@herrie.org>
Date: Thu, 28 Dec 2017 16:35:09 +0100
Subject: [PATCH] common.mk: Drop com.cyanogenmod.keyhandler

Signed-off-by: Herman van Hazendonk <github.com@herrie.org>
---
 common.mk | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/common.mk b/common.mk
index 4ae6fd8..f5825bd 100644
--- a/common.mk
+++ b/common.mk
@@ -19,8 +19,7 @@ DEVICE_PACKAGE_OVERLAYS += $(LOCAL_PATH)/overlay
 
 # Keyhandler
 PRODUCT_PACKAGES += \
-    ConfigPanel \
-    com.cyanogenmod.keyhandler
+    ConfigPanel
 
 PRODUCT_SYSTEM_SERVER_JARS += com.cyanogenmod.keyhandler
 
-- 
2.13.2.windows.1

